docs/core/diagnostics/cli-tools/dotnet-dump.md

```diff
@@ -0,0 +1,141 @@
+---
+title: dotnet-dump - .NET Core
+description: Installing and using the dotnet-dump command-line tool.
+author: sdmaclea
+ms.author: stmaclea
+ms.date: 08/21/2019
+---
+# Dump collection and analysis utility (`dotnet-dump`)
+
+The `dotnet-dump` CLI global tool is a way to collect and analyze Windows and Linux dumps all without any native debugger involved like `lldb` on Linux. This tool is important on platforms like Alpine Linux where a fully working `lldb` isn't available. The `dotnet-dump` tool will allow you to run SOS commands to analyze crashes and the GC, but it isn't a native debugger so things like displaying native stack frames aren't supported.
+
+> [!NOTE]
+> `dotnet-dump` is not supported on macOS.
+
+## Installing `dotnet-dump`
+
+To install the latest release version of the `dotnet-dump` [NuGet package](https://www.nuget.org/packages/dotnet-dump):
+
+```bash
+dotnet tool install -g dotnet-dump
+```
+
+For details and other options, see [Installing the diagnostics tools](installing.md).
+
+> [!NOTE]
+> At the time of this writing, the `dotnet-dump` tool has not been released. The prerelease version can be installed using:
+>
+> ```
+> dotnet tool install --global dotnet-dump --version 3.0.0-preview8.19412.1
+> ```
+>
+
+## Using `dotnet-dump`
+
+The next step is to collect a dump. This step can be skipped if a core dump has already been generated. The operating system or the .NET Core runtime's built-in [dump generation feature](https://github.com/dotnet/coreclr/blob/master/Documentation/botr/xplat-minidump-generation.md#configurationpolicy) can each create core dumps.
```

Comment:
You start with the next step. So, what's the first step about using the tool?

Reply:
I saw it as the install...

I'll clean it up.